src/mlmodule/torch/runners.py

```diff
@@ -283,6 +289,8 @@ class TorchTrainingRunner(
         TorchTrainingOptions,
     ]
 ):
+    loggers_factory: Optional[Callable[[Engine, Engine, Engine], None]] = None
```

Comment:
I would be in favor of keep the runners API static as much as possible.

Would there be a problem putting loggers_factory in the options rather that adding a new argument ?

Reply:
Shouldn't be a problem. I'll make the change.

Reply:
See https://github.com/LSIR/mlmodule/pull/161/commits/f2abde6d27c3097ae53ce069b6fb6a43f98e0b07

Reply:
Ok perfect !